README.md

```diff
@@ -1,2 +1,176 @@
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Live Demo  - click [TriumphTribute](https://mpdaddy.github.io/capstone_prize-giving_day/)
+
+## 💻 Getting Started <a name="getting-started"></a>
+Bellow is how you may want to use this project
+
+### Prerequisites
+
+In order to run this project you need:
+
+terminal and your favourite editor.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:MPDADDY/capstone_project.git 
+```
+### Install
+
+Install this project by: 
+cloning it to your local machine.
+using the command: [git clone] and pasting https://github.com/MPDADDY/capstone_prize-giving_day.git link.
+
+
+Example command:
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+make sure you are in the right directory that contains your project.Run this command: 'code .'
+and the project will open in your favourite text editor.
+
+git clone git@github.com:MPDADDY/capstone_project.git
+
+Example command:
+
+
+
+### Deployment
+
+- click [TriumphTribute](https://mpdaddy.github.io/capstone_prize-giving_day/) for the deployed site.
+
+### Original design for the project
+
+- click [TriumphTribute design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) for the project design on Behance.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> This project has been coded by Anold.
+
+👤 **Author**
+
+- GitHub: [@mpdaddy](https://github.com/MPDADDY)
+- Twitter: [@anoldmwawasi](https://twitter.com/anold_mwawasi)
+- LinkedIn: [anoldmwawasi](https://www.linkedin.com/in/anold-mwawasi/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[animations]**
+Transition will be added to some elements to create some animation. 
+- [ ] **[new_feature_2]**
+This will be provided later.
+- [ ] **[new_feature_3]**
+This will be provided later.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free start any issue on [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+Comments and suggestions from readers will be highly appreciated.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+I would like to thank Cindi Shin for the design.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The first reviewer forgot to mention it but you should fix the broken MIT link, having broken links in the README is not very professional 🙃 🙃 

<img width="714" alt="Screenshot 2023-08-01 at 16 16 07" src="https://github.com/MPDADDY/capstone_prize-giving_day/assets/74501523/7721bef0-4353-47ec-be5e-e9e17b6cf8ac">